frontend/src/components/BtoItemCard.tsx

```diff
@@ -1,17 +1,29 @@
-import { Markup } from "interweave";
+import {Markup} from "interweave";
 import {BtoDisplayItem} from "../model/BtoDisplayItem";
+import "../style/BtoItemCard.css"
+import {useNavigate} from "react-router-dom";
 
-type BtoItemsCardProps = {
+type BtoItemCardProps = {
     btoDisplayItem: BtoDisplayItem;
 }
 
-export default function BtoItemCard({btoDisplayItem}: BtoItemsCardProps) {
-
-    return(
-        <div className={"bto-item-card"}>
-            <h1>{btoDisplayItem.title1}</h1>
-            <h2>{btoDisplayItem.title2}</h2>
-            <Markup content = {btoDisplayItem.description} />
+export default function BtoItemCard({btoDisplayItem}: BtoItemCardProps) {
+    const navigate = useNavigate();
+    if (btoDisplayItem.category !== "action")
+        return (
+            <div className={"bto-item-card"}>
+                <h1>{btoDisplayItem.title1}</h1>
+                <h2>{btoDisplayItem.title2}</h2>
+                <Markup content={btoDisplayItem.description}/>
+            </div>
+        )
+    return (
+        <div className={"bto-item-action-card"}>
+            <div className={"action-title1"}>Aktion: {btoDisplayItem.title1}</div>
+            <div className={"action-title2"}> {btoDisplayItem.title2}</div>
+            <p className={"action-status"}>Status: {btoDisplayItem.status}
+                <button onClick={() => navigate(`btoItem/${btoDisplayItem.id}`)}>Details</button>
+            </p>
         </div>
```

Comment:
HIer würde ich nochmal bisschen refactoren.
Ich würde vielleicht zwei verschiedene Komponenten bauen die, die verschiedenen Items anzeigen.
Ich glaube ich würde auch eher mit einem ? (ternary operator) arbeiten anstatt einem if

Reply:
Den tenären Operator habe ich nun eingesetzt und es sieht tatsächlich übersichtlicher aus. 
Den Vorteil von der Ausgliederung in verschiedene Komponenten sehe ich hier im Moment noch nicht, ich lasse das im Moment mal noch so